godot-cpp fail to build on Windows due to encoding issue

When building godot-jolt, I encountered issue:
```
E:\code\godot-jolt\build\windows-msvc-x64\External\Build\godot-cpp\gen\include\godot_cpp/classes/rich_text_label.hpp(110): error C2001: newline in constant
```

Then in the code I see:
```
	void push_list(int32_t level, RichTextLabel::ListType type, bool capitalize, const String &bullet = "•");
```

Knowing this must be an encoding issue. I resaved that cached `rich_text_label.hpp` file as UTF-8 with BOM, and the build succeeded.

I believe this is relevant to the machine's Windows locale settings, and can be solved by forcing UTF somewhere. Knowing nothing about cmake language, I'm posting this issue here. If desirable, please let me know where to forward this issue to.

--- a/cmake/GodotJoltExternalGodotCpp.cmake
+++ b/cmake/GodotJoltExternalGodotCpp.cmake
@@ -19,7 +19,7 @@ set(editor_definitions
 
 gdj_add_external_library(godot-cpp "${configurations}"
 	GIT_REPOSITORY https://github.com/godot-jolt/godot-cpp.git
-	GIT_COMMIT c0dd7eeada027006d4ebed62a44ac9378d4147d3
+	GIT_COMMIT 43d02f2d7c0a051d7134904168ce92615d11713e
 	LANGUAGE CXX
 	OUTPUT_NAME godot-cpp
 	INCLUDE_DIRECTORIES

